cron/internal/data/projects.csv

```diff
@@ -635954,6 +635955,7 @@ github.com/jayway/facebook-test-java-api,num_dependents_deps.dev:0
 github.com/jayway/forest,num_dependents_deps.dev:24
 github.com/jayway/getsandbox-express,num_dependents_deps.dev:0
 github.com/jayway/jax-rs-hateoas,num_dependents_deps.dev:12
+github.com/json-path/JsonPath,
```

Comment:
`github.com/json-path/JsonPath` is already on the list.

Reply:
this error might just be from the other one sorry. but `make add-projects` will move this to the right place alphabetically.